agent/api/container/container.go

```diff
@@ -110,6 +110,8 @@ type Container struct {
 	Image string
 	// ImageID is the local ID of the image used in the container
 	ImageID string
+	// ImageDigest is the sha-256 digest of the container image as pulled from the repository
+	ImageDigest string
```

Comment:
this may require a statefile version update

Reply:
will update the statefile version.